superset/models/core.py

```diff
@@ -761,8 +761,11 @@ def wrapper(*args, **kwargs):
             post_data = request.form or {}
             d.update(post_data)
             d.update(kwargs)
-            slice_id = d.get('slice_id', 0)
             try:
+                slice_id = d.get('slice_id') 
```

Comment:
it's better to keep try blocks as nimble as possible, 765:766 are safe (`dict.get` won't throw key errors if the key doesn't exist it'll just return `None`) and can be executed outside the try block.